pom.xml

```diff
@@ -42,7 +42,7 @@
   </organization>
 
   <properties>
-    <hadoop.version>2.7.3</hadoop.version>
+    <hadoop.version>2.8.0</hadoop.version>
```

Comment:
It is OK to use Hadoop's implementation, just try to not directly refer to any object in `LdapAuthenticationHandler `, all you need to do is to set proper parameters to `FilterHolder`. And you can build Livy against Hadoop 2.8 your own, then you will have Ldap supported transparently.